app/views/idv/doc_auth/verify.html.erb

```diff
@@ -40,7 +40,20 @@
       class: 'btn btn-link',
     ) %>
   </div>
-  <%= "#{t('doc_auth.forms.ssn')}: #{flow_session[:pii_from_doc][:ssn]}" %>
+  <%= t('doc_auth.forms.ssn') %>
+  <div class='sm-col sm-col-8 px0 mt0 mb3'>
+    <%= tag.input value: flow_session[:pii_from_doc][:ssn],
+                  class: 'block col-12 field password ssn ssn-toggle bg-white',
+                  aria: { label: t('doc_auth.forms.ssn'), invalid: false, required: false },
+                  readonly: true,
+                  maxlenth: 11,
```

Comment:
Typo:

```suggestion
                  maxlength: 11,
```

Since it's readonly, we may not need these constraints at all, though maybe they're assumed to exist with the SSN toggle JavaScript.